packages/contracts-bedrock/contracts/test/L2StandardBridge.t.sol

```diff
@@ -16,10 +17,6 @@ import { OptimismMintableERC20 } from "../universal/OptimismMintableERC20.sol";
 contract L2StandardBridge_Test is Bridge_Initializer {
     using stdStorage for StdStorage;
 
-    function setUp() public override {
```

Comment:
Why is this removed?

Reply:
The super function is already public, so there's no need to override it with a Function That just calls it again

https://github.com/ethereum-optimism/optimism/blob/9f08dcbb93e5b2fba6ac76072048b735f429cbd9/packages/contracts-bedrock/contracts/test/CommonTest.t.sol